notebooks/official/custom/sdk-custom-image-classification-batch.ipynb

```diff
@@ -78,7 +81,7 @@
         "In this tutorial, you learn to use Vertex AI Training to create a custom trained model and use Vertex AI Batch Prediction to do a batch prediction on the trained model.\n",
         "\n",
         "\n",
-        "create a custom-trained model from a Python script in a Docker container using the Vertex AI SDK for Python, and then do a prediction on the deployed model by sending data. Alternatively, you can create custom-trained models using `gcloud` command-line tool, or online using the Cloud Console.\n",
+        "Create a custom-trained model from a Python script in a Docker container using the Vertex AI SDK for Python, and then do a prediction on the deployed model by sending data. Alternatively, you can create custom-trained models using `gcloud` command-line tool, or online using the Cloud Console.\n",
```

Comment:
Vertex AI SDK for Python

Reply:
@kittyabs could you please elaborate this comment. Which particular change we need to make?